src/components/Teaser/Teaser.tsx

```diff
@@ -1,104 +1,182 @@
+                                            matchingIndicatorPercentage
+                                        )}
+                                        fill="none"
+                                        strokeWidth="3"
+                                        strokeLinecap="round"
+                                        strokeDasharray={`${(matchingIndicatorPercentage / 100) * 56}, 56`}
+                                    />
+                                </svg>
+                            )}
+                    </div>
+
+                    <div {...elem('row')}>
+                        {timestamp && (
+                            <Text inline title={timestamp} {...elem('caption')}>
+                                {timestamp}
+                            </Text>
+                        )}
+                        {typeof isVisited !== 'undefined' && (
+                            <div
+                                {...elem('visibility', { isVisited })}
+                                aria-label="Visibility indicator"
+                            />
+                        )}
+                    </div>
+                </div>
+                {subtitle && (
+                    <Text inline title={subtitle} {...elem('subtitle')}>
+                        {subtitle}
                     </Text>
                 )}
-            </div>
-            <div {...elem('line')}>
-                {subTitle && subTitleElement}
-                {!subTitle && details && detailsElement}
-            </div>
-            <div {...elem('line')}>
-                {subTitle && details && detailsElement}
-                {statuses && statuses.length > 0 && (
-                    <div {...elem('statuses')}>
-                        {statuses.map(({ label, tooltip }, index) => (
-                            <span key={label}>
-                                <Tooltip content={tooltip}>
-                                    <Text inline context="neutral" size="small">
-                                        {label}
-                                    </Text>
-                                </Tooltip>
-                                {index < statuses.length - 1 && (
-                                    <Text inline context="neutral" size="small">
-                                        {' & '}
-                                    </Text>
-                                )}
-                            </span>
-                        ))}
+                {primaryInfo && <InfoComponent text={primaryInfo.text} href={primaryInfo.href} />}
+                {secondaryInfo && (
+                    <InfoComponent text={secondaryInfo.text} href={secondaryInfo.href} />
+                )}
+                {tercearyInfo && (
+                    <InfoComponent text={tercearyInfo.text} href={tercearyInfo.href} />
+                )}
+                {sourceInfo && <InfoComponent text={sourceInfo.text} icon={sourceInfo.icon} />}
```

Comment:
wouldn't source be clickable e.g. in Jobfeed? How will that work?

Reply:
You can pass a href as well to the source if there is a need to be a Link